tests/integration-tests/tests/runtime_bake/test_runtime_bake.py

```diff
@@ -13,35 +13,30 @@
 import pytest
 from remote_command_executor import RemoteCommandExecutor
 
-from tests.common.schedulers_common import get_scheduler_commands
 from tests.common.utils import retrieve_latest_ami
 
 
-@pytest.mark.skip(reason="Temporarily disable this test")
-@pytest.mark.dimensions("eu-west-3", "c5.xlarge", "alinux2", "torque")
-@pytest.mark.dimensions("us-east-2", "c5.xlarge", "centos7", "sge")
-@pytest.mark.dimensions("us-east-2", "c5.xlarge", "centos8", "sge")
-@pytest.mark.dimensions("eu-west-1", "c5.xlarge", "ubuntu1804", "sge")
+@pytest.mark.dimensions("eu-west-2", "c5.xlarge", "centos8", "slurm")
+@pytest.mark.dimensions("us-gov-east-1", "c5.xlarge", "ubuntu1804", "slurm")
 @pytest.mark.dimensions("us-gov-west-1", "c5.xlarge", "ubuntu1804", "sge")
-@pytest.mark.dimensions("us-east-1", "m6g.xlarge", "ubuntu1804", "sge")
-@pytest.mark.dimensions("eu-west-1", "m6g.xlarge", "alinux2", "slurm")
+@pytest.mark.dimensions("eu-west-2", "c5.xlarge", "alinux2", "torque")
+@pytest.mark.dimensions("eu-west-2", "c5.xlarge", "centos7", "sge")
+@pytest.mark.dimensions("eu-west-2", "c5.xlarge", "ubuntu2004", "torque")
+# @pytest.mark.dimensions("us-east-1", "m6g.xlarge", "alinux2", "slurm")
+# @pytest.mark.dimensions("us-east-1", "m6g.xlarge", "ubuntu1804", "sge")
 @pytest.mark.usefixtures("instance", "scheduler")
 def test_runtime_bake(scheduler, os, region, pcluster_config_reader, clusters_factory, test_datadir, architecture):
     """Test cluster creation with runtime bake."""
-    # remarkable AMIs are not available for ARM yet
-    ami_type = "remarkable" if architecture == "x86_64" else "official"
+    # remarkable AMIs are not available for ARM yet and disable centos7,8 remarkable AMIs
```

Comment:
From what you said offline we're not using remarkable AMIs for centos 7 or 8 for x86_64 for two reasons:
* FPGA ami’s volume size is not enough, so we disable testing it
* FPGA ami installs epel, it is not suitable for our case to test it

Can you add this information to the comment?